server/src/inmantals/server.py

```diff
@@ -255,7 +258,15 @@ async def dispatch_method(self, id, method, params):
 
         return result
 
-    async def send_show_message(self, type, message):
+    async def send_show_message(self, type: int, message: str):
+        """
+        Show a pop up message to the user, type gives the level of the message, message is the content
+        Available types are:
+         * 1: Error
```

Comment:
Maybe add an Enum?

Reply:
(`inmantals.lsp_types` is the place to add it)

Reply:
Did it in https://github.com/inmanta/vscode-inmanta/pull/33/commits/939de709d3b0da790beb077a77b621e46ae04848